index.html

```diff
@@ -14,8 +14,14 @@
       <a href="welcome" class="head">Welcome</a>
       <nav class="menu-bar">
         <img src="images/Normal Button/Enabled.png" alt="">
+        </nav>
       </nav>
     </header>
+    <ul class="nav-menu">
+      <li><a href="#portfolio">Portfolio</a></li>
+      <li><a href="#about">About</a></li>
+      <li><a href="#contact">Contact</a></li>
+    </ul>
     <section class="head-section">
     <div class="hey-there">
 <h1 class="hey-text">Hey there. I’m Mariane <br>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to `Home Page` and also `Navigation` elements like [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=FM3eIBvsAszGSaO5-0)🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/221366847-883dddd8-10a1-4da9-ab05-997514b12824.png)